docs/main_text.md

```diff
@@ -426,6 +426,7 @@ We offer two kinds of refine.bio compendia: [normalized compendia](#normalized-c
 refine.bio normalized compendia are comprised of all the samples from a species that we were able to process, aggregate, and normalize.
 Normalized compendia provide a snapshot of the most complete collection of gene expression that refine.bio can produce for each supported organism.
 We process these compendia in a manner that is different from the options that are available via the web user interface.
+Note that [submitter processed](#submitter-processed-submitter-processed-badge) samples that are available through the web user interface are omitted from normalized compendia because [these samples can introduce unwanted technical variation](https://github.com/AlexsLemonade/refinebio/issues/2114).
```

Comment:
```suggestion
Note that [submitter processed](#submitter-processed--) samples that are available through the web user interface are omitted from normalized compendia because [these samples can introduce unwanted technical variation](https://github.com/AlexsLemonade/refinebio/issues/2114).
```

Linking to `#submitter-processed--` seems to make the jump to the submitter processed section.